jekyll/_cci2/roles-and-permissions-overview.adoc

```diff
@@ -291,6 +291,36 @@ The table below shows the permissions associated with each CircleCI organization
 ^|
 ^|
 
+| *Releases*
+|
+|
+|
+
+^| Create release integration
+^| icon:check-circle[]
+^|
+^|
+
+^| Delete release integration
+^| icon:check-circle[]
+^|
+^|
```

Comment:
These permissions [will be changed](https://circleci.atlassian.net/wiki/spaces/DP/pages/6963200001/REL-DR-020+Open+Preview+User+Permissions+Rules) before we launch open preview + we decided to call these "environments" for now:
```suggestion
| *Releases*
|
|
|

^| Create release environment
^| icon:check-circle[]
^| icon:check-circle[]
^|

^| Delete release environment
^| icon:check-circle[]
^| icon:check-circle[]
^|
```

Reply:
ahh true, I forgot it was accepted